src/index.ts

```diff
@@ -117,4 +121,25 @@ const legacyPlugin: JupyterFrontEndPlugin<void> = {
   }
 };
 
-export default [plugin, legacyPlugin];
+/**
+ * The notebook cell executor.
+ */
+const executor: JupyterFrontEndPlugin<void> = {
+  id: 'jupyterlab-myst:executor',
```

Comment:
We add another plugin to allow markdown cells to be executed.

Reply:
so 2 plugins are installed now? and the both work independently?
i.e. the other plugin takes care of Markdown cell rendering whilst this one takes care of execution post render? or are we also rendering here *when* there is an inline expression?

Reply:
A single JupyterLab extension can install multiple plugins. Here we provide the "execute inline expressions and store results" using this `:executor` plugin. The renderering is still exclusively performed by the `:plugin` plugin.

Reply:
It does mean that you can turn parts of it off at a time, which is quite nice!